<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchiveTableDialog</class>
 <widget class="QDialog" name="ArchiveTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="tbAdd">
         <property name="toolTip">
          <string>Add new row</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="shvsm.qrc">
           <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbDel">
         <property name="toolTip">
          <string>Remove current row</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="shvsm.qrc">
           <normaloff>:/image/minus.png</normaloff>:/image/minus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbPost">
         <property name="toolTip">
          <string>Save data</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="shvsm.qrc">
           <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbCancel">
         <property name="toolTip">
          <string>Cancel changes</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="shvsm.qrc">
           <normaloff>:/image/cancel.png</normaloff>:/image/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="shvsm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbAdd</sender>
   <signal>clicked()</signal>
   <receiver>ArchiveTableDialog</receiver>
   <slot>slotAddRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbDel</sender>
   <signal>clicked()</signal>
   <receiver>ArchiveTableDialog</receiver>
   <slot>slotDelRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbPost</sender>
   <signal>clicked()</signal>
   <receiver>ArchiveTableDialog</receiver>
   <slot>slotSaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ArchiveTableDialog</receiver>
   <slot>slotCancelChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAddRow()</slot>
  <slot>slotDelRow()</slot>
  <slot>slotSaveChanges()</slot>
  <slot>slotCancelChanges()</slot>
 </slots>
</ui>
